stable/jenkins/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: jenkins
 home: https://jenkins.io/
-version: 1.7.3
```

Comment:
@mogaal  Previous chart version was already 1.7.3. What would you like me to do?